Source/KeyboardManager.swift

```diff
@@ -5,109 +5,160 @@
 
 import UIKit
 
-public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent, KeyboardManagerEventData) -> Void
+public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent) -> Void
 
 public enum KeyboardManagerEvent {
-    case willShow
-    case didShow
-    case willHide
-    case didHide
-}
+    case willShow(KeyboardManagerEvent.Data)
+    case didShow(KeyboardManagerEvent.Data)
+    case willHide(KeyboardManagerEvent.Data)
+    case didHide(KeyboardManagerEvent.Data)
 
-public struct KeyboardManagerEventData {
     public struct Frame {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/25a2ec034b913c48e066e9db453abead1303374d/Source/KeyboardManager.swift#L16"></span>Public declarations should be documented.</td>
    </tr>
  </tbody>
</table>